src/pages/api/oauth/callback.ts

```diff
@@ -23,21 +23,18 @@ const handler: Handler = async (
       `${request.query.code}`,
     );
 
-    const decodedAccessToken = await oauth2Client.verifyJWT<AccessToken>(
-      access_token,
-      config.connectJwtAlgorithm,
-    );
+    const decodedToken = await decryptVerifyAccessToken(access_token);
```

Comment:
Any reasons why you changed the name to `decodedToken`? Shouldn't `decryptVerifyAccessToken` return an access token?

Reply:
Hmm good point. I can't see why I changed that, indeed.